resources/views/questions/show.blade.php

```diff
@@ -114,7 +114,7 @@ class="{{empty($question->hint_image)? 'hidden': ''}}">
         
         var xml = Blockly.Xml.workspaceToDom(Blockly.getMainWorkspace());
         var blocks = encodeURIComponent(xml.innerHTML);
-        var image = $('.game')[0].contentDocument.getElementById('display').toDataURL('image/png');
+        var image = $('.game')[0].contentDocument.getElementById('display').toDataURL('image/jpeg',1.0);
 
```

Comment:
yes you are right there is no need to change the type
when we exported it using png we had trouble that the background is transparent 
but after filling background with white there is no need